data/themes/elarun/Main.qml

```diff
@@ -200,18 +200,24 @@ Rectangle {
                         }
                     }
 
-                    Text {
-                        id: time_label
+                    Rectangle {
+                        id: container
+                        property date dateTime: new Date()
+                        property color color: "#0b678c"
```

Comment:
this doesn't feel right, Rectangle already has a color property.
just set the color property on Text below like it was before.